controllers/candidatesController.js

```diff
@@ -98,29 +91,33 @@ module.exports = {
       other_aspects: req.body.other_aspects,
       work_culture_preferences: req.body.work_culture_preferences,
     };
-
-    Candidate.findByIdAndUpdate(candidateId, { $set: candidateParams })
-      .then(candidate => {
-        res.locals.redirect = `/user/${req.app.locals.user._id}`;
-        res.locals.candidate = candidate;
-        next();
+    //has to be findOneAndUpdate bc it works with mongoosastic
+    Candidate.findOneAndUpdate({_id: candidateId},
+      {$set: candidateParams},
+      {new: true})
+    .then(candidate => {
+      res.locals.redirect = `/candidates/${candidateId}`;
```

Comment:
We changed the redirect for a reason. That shouldn't be changed

Reply:
hey.. sorry about the controller.. We actually didnt change anything there but might because when we do the pull rebase, there was some mistake with merging code.. I will fix that. Thanks a lot for your review.